### PR TITLE
Text change

### PR DESCRIPTION
**Pull request by Jxmes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   
   <div id="vertically-align"></div>
   </div>
-  <p id="spooky">deep web: activated.<br>bru-mobile firing up.</p>
+  <p id="spooky">deep web: activated.<br>BRU-MOBILE FIRING UP.</p>
```
Comment (A1fus) [inline index.html]: Is `deep web: activated` caps aswell?

Comment (Jxmes) [inline index.html]: no

Comment (A1fus) [inline index.html]: Why?

Comment (Jxmes) [inline index.html]: dunno

Comment (A1fus) [inline index.html]: pLEASE MAKE ALL CAPS OR NO PULL

**Comment by A1fus** [comment]
Thanks :+1:

**Pull request merged**